tests/test_helpers.py

```diff
@@ -0,0 +1,42 @@
+import torch
+from transformers import BertForPreTraining, AutoTokenizer
+
+
+@staticmethod
+def get_graph_module_for_conv_model(model, inputs):
+    return torch.fx.symbolic_trace(model)
```

Comment:
lets remove this if we are not using it anywhere.

Reply:
Yup you're right. That was an oversight on my part